app/views/waste_carriers_engine/start_forms/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="text">
+  <%= form_for(@start_form) do |f| %>
+    <%= render("waste_carriers_engine/shared/errors", object: @start_form) %>
+
+    <h1 class="heading-large"><%= t(".heading") %></h1>
+
+    <div class="form-group <%= "form-group-error" if @start_form.errors[:start_option].any? %>">
+      <fieldset id="start_option">
+        <legend class="visuallyhidden">
+          <%= t(".heading") %>
+        </legend>
+
+        <% if @start_form.errors[:start_option].any? %>
+          <span class="error-message"><%= @start_form.errors[:start_option].join(", ") %></span>
+        <% end %>
+
+        <div class="multiple-choice">
+          <%= f.radio_button :start_option, 'new' %>
```

Comment:
Can we use the START_OPTIONS constant at all here? :bowtie:

Reply:
Oh yes! I forgot to make that fix! Thanks for pointing that out :D

Reply:
```suggestion
          <%= f.radio_button :start_option, WasteCarriersEngine::StartForm::NEW %>
```